app/src/main/java/com/google/android/apps/flexbox/FlexItem.java

```diff
@@ -146,6 +156,8 @@ public static FlexItem fromFlexView(View view, int index) {
         flexItem.paddingStart = ViewCompat.getPaddingStart(view);
         flexItem.paddingEnd = ViewCompat.getPaddingEnd(view);
         flexItem.paddingBottom = view.getPaddingBottom();
+        flexItem.minWidth = lp.minWidth;
```

Comment:
Also could you follow the width/height fields as 

flexItem.minWidth = Util.pixelToDp(view.getContext, lp.minWidth);
flexItem.minHeight = Util.pixelToDp(view.getContext, lp.minHeight);